standalone-metastore/src/main/sql/mysql/hive-schema-3.0.0.mysql.sql

```diff
@@ -1100,6 +1101,62 @@ CREATE TABLE `SCHEMA_VERSION` (
   KEY `UNIQUE_VERSION` (`SCHEMA_ID`, `VERSION`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+CREATE TABLE `SCHEMA_VERSION_STATE` (
+  `SCHEMA_VERSION_STATE_ID` BIGINT NOT NULL,
+  `SCHEMA_VERSION_ID` BIGINT NOT NULL,
+  `STATE_ID` BIGINT NOT NULL,
+  `SEQUENCE` INT NOT NULL,
+  `TIMESTAMP` BIGINT NOT NULL,
+  `DETAILS` BLOB,
+  PRIMARY KEY (SCHEMA_VERSION_ID, STATE_ID, SEQUENCE),
+  UNIQUE KEY (id)
+);
+CREATE TABLE `SCHEMA_FIELD` (
```

Comment:
SCHEMA_VERSION has a CD_ID field that connects it to a set of columns.  This appears to duplicate that.  Or does it track something else?